packages/menu/src/Menu/Menu.js

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Button } from 'pcln-design-system'
+import { ChevronDown } from 'pcln-icons'
+import Popover from 'pcln-popover'
+import MenuList from '../MenuList'
+
+const LinkButton = styled(Button)`
+  display: flex;
+  align-items: center;
+  padding: 8px;
+`
+
+function Menu({ buttonText, color, height, size, children, ...props }) {
+  const MenuContent = ({ handleClose }) => (
+    <MenuList
+      id='menu-container'
```

Comment:
Since there's the possibility that there might be more than menu on the page, we probably need to add a prefix to this id. We can't have repeated ids on the page.